fernet.go

```diff
@@ -63,7 +64,7 @@ func decodedLen(n int) int {
 
 // if msg is nil, decrypts in place and returns a slice of tok.
 func verify(msg, tok []byte, ttl time.Duration, now time.Time, k *Key) []byte {
-	if len(tok) < 1 || tok[0] != version {
+	if len(tok) < 1+uint64Bytes || tok[0] != version {
```

Comment:
```suggestion
	if len(tok) <= uint64Bytes || tok[0] != version {
```

Reply:
@kayrus why?

`1+uint64Bytes` mean the minimal header length we need: 1 is the version byte and `uint64Bytes` are the timestamp followed. `<= uint64Bytes` doesn't have such meaning.

Reply:
`len(tok) < 1+uint64Bytes` equals to `len(tok) <= uint64Bytes` and the last is more readable
see https://play.golang.org/p/ee81FrVXUI3

Reply:
I know they are the same, but I disagree that `<= uint64Bytes` is more readable, as the reason I stated.

Reply:
ok, I don't insist.